File: backend/roles.py
ROLE_ADMIN = "admin"
ROLE_HR = "hr"
ROLE_CANDIDATE = "candidate"

ALL_ROLES = (ROLE_ADMIN, ROLE_HR, ROLE_CANDIDATE)

ROLE_LABELS = {
    ROLE_ADMIN: "Administrator",
    ROLE_HR: "HR Manager",
    ROLE_CANDIDATE: "Candidate",
}


def is_valid_role(value: str) -> bool:
    """Return True when the supplied role exists in the system."""
    return value in ALL_ROLES

File: hrms-project/src/constants/roles.js
export const ROLE_ADMIN = 'admin';
export const ROLE_HR = 'hr';
export const ROLE_CANDIDATE = 'candidate';

export const ROLE_LABELS = {
  [ROLE_ADMIN]: 'Administrator',
  [ROLE_HR]: 'HR Manager',
  [ROLE_CANDIDATE]: 'Candidate',
};

export const DASHBOARD_ROUTE_BY_ROLE = {
  [ROLE_ADMIN]: '/admin-dashboard',
  [ROLE_HR]: '/hr-dashboard',
  [ROLE_CANDIDATE]: '/candidate-dashboard',
};
